test/unit/alphabet/aminoacid/aminoacid_conversion_integration_test.cpp

```diff
@@ -10,23 +10,23 @@
 #include <meta/meta.hpp>
 
 #include <seqan3/alphabet/aminoacid/all.hpp>
-#include <seqan3/core/metafunction/template_inspection.hpp>
+#include <seqan3/core/type_list.hpp>
+#include <seqan3/test/testing_list.hpp>
 
 using namespace seqan3;
 
 template <typename T>
 class aminoacid_conversion : public ::testing::Test
 {};
 
-using aminoacid_types2 = meta::list<aa20, aa27>; // needed for some tests
-using aminoacid_types = detail::transfer_template_args_onto_t<aminoacid_types2, ::testing::Types>;
+using aminoacid_types = type_list<aa20, aa27>; // needed for some tests
 
-TYPED_TEST_CASE(aminoacid_conversion, aminoacid_types);
+TYPED_TEST_CASE(aminoacid_conversion, as_testing_list<aminoacid_types>);
```

Comment:
So `detail::transfer_template_args_onto_t<aminoacid_types2, ::testing::Types>` would also have worked right?

Reply:
Yeah would have, I can revert to this. I thought it would be easier to read, but I can just use `transfer_template_args_onto_t`

Reply:
It's just that for me it is just another name that is explicitly introduced in the testing environment. So I would prefer the  transfer_template_... thing as it is documented and we only need it at some few places.